app/components/emoji_picker/index.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {createSelector} from 'reselect';
+
+import {getCustomEmojisByName} from 'mattermost-redux/selectors/entities/emojis';
+
+import {getTheme} from 'app/selectors/preferences';
+import {CategoryNames, Emojis, EmojiIndicesByCategory} from 'app/utils/emojis';
+
+import EmojiPicker from './emoji_picker';
+
+const categoryToI18n = {
+    activity: {
+        id: 'mobile.emoji_picker.activity',
```

Comment:
Aren't this strings already in the localization files? Looks to me that they are